### PR TITLE
Fix lint problems

### LINKED ISSUE #171: Lint errors causes build failure
When building with "gradlew build". There are some new lint errors.

1. 
A removed activity is still present in the manifest. OpenHABInfoActivity which was changed to a fragment in #161.

2. 
The log TAG exceeds 23 characters. There is a new lint rule that checks for long log tags to comply with the log api: http://developer.android.com/reference/android/util/Log.html.

1 can be corrected by removing the entry from the manifest. In the same time maybe changing the log tag to current class name?

2 can be "solved" by adding
lintOptions {disable 'LongLogTag'} to build.gradle.
But imo it is better to fix the root cause of the problem: Shortening the log tag. 
The following two classes have to long log tag:
ConnectivityChangeReceiver and NotificationDeletedBroadcastReceiver

Any suggestions how to shorten the tag (or class as well)?

On another note. There is a lot of classes prepended with OpenHAB, that can be shortened to OH to save some characters ;-)

### PR DESCRIPTION
**Pull request by hazzeh**
Fixes #171 
Also specify android wearable support version
This will also fix the "AAPT: No resource found that matches the given name" shown in https://openhab.ci.cloudbees.com/job/HABDroid/98/console

### CODE REVIEW

File: mobile/src/main/java/org/openhab/habdroid/core/ConnectivityChangeReceiver.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 import android.util.Log;
 
 public class ConnectivityChangeReceiver extends BroadcastReceiver {
-    private static final String TAG = "ConnectivityChangeReceiver";
+    private static final String TAG = ConnectivityChangeReceiver.class.getSimpleName();
```
Comment (hazzeh) [inline mobile/src/main/java/org/openhab/habdroid/core/ConnectivityChangeReceiver.java]: Maybe this should be done in all classes for consistency?

Comment (zacharyyy) [inline mobile/src/main/java/org/openhab/habdroid/core/ConnectivityChangeReceiver.java]: I think so. The lint warnings about the TAG being too long will go away and if you refactor the class name, the tag will be updated too.

Comment (hazzeh) [inline mobile/src/main/java/org/openhab/habdroid/core/ConnectivityChangeReceiver.java]: I've pushed an updated commit now.
All classes that uses logging has been updated.

**Comment by digitaldan** [comment]
Thank for the PR , I'll test this asap.

**Pull request merged**